src/modules/main/sidebar/sidebar-panel/SidebarPanel.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react'
+
+import { Logo } from 'ui';
+import Cancel from '@material-ui/icons/Cancel';
```

Comment:
Material ui things should be moved between react import and Logo from UI - remember about enter between both

Reply:
Formatting has been improved